docs/people/started.md

```diff
@@ -0,0 +1,9 @@
+---
+id: started
+title: Getting Started
+sidebar_label: Getting Started
+---
+
+All [Deskera](https://www.deskera.com/books/) endpoints are secured and can only be reached after a successful authentication. Deskera APIs require a valid token to communicate. This token should be passed as `x-access-token` in the request header while interacting Deskera APIs. To generate a valid token, the user can trade-off username and password with Authentication API. Authentication API will return accessToken in the response body which can be used to communicate with Deskera APIs.
```

Comment:
i hope you see it is pointing to books url

Reply:
done, please check